src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -515,4 +515,15 @@ code-inner {
 }
 .code-removed {
     background-color: #ffeef0;
+}
+
+.deprecated {
+    text-decoration: line-through;
+}
+
+.documentation {
```

Comment:
These styles look very similar to code line default styles, we shouldn't repeat them.

Reply:
Sure. I will update them.